heron/scheduler-core/src/java/com/twitter/heron/scheduler/utils/SchedulerUtils.java

```diff
@@ -167,7 +202,7 @@ private SchedulerUtils() {
       Config config,
       Config runtime,
       int containerIndex,
-      List<String> ports) {
+      Map<ExecutorPort, String> ports) {
```

Comment:
Please update the javadoc for `ports`. 
Also one of these two methods can be deleted, `getExecutorCommand` and `executorCommand`. I am not sure if the intent is to consume `<String, Integer>` type map in one of the methods.

Reply:
I will delete the executorCommand method.  It was my intention to pass the port map where the value is a string since some schedulers specify their ports as a string such as the Marathon Scheduler.